R/hankel.R

```diff
@@ -43,7 +35,15 @@ gevp.hankel.old <- function(cf, t0, deltat = 1, n, N, id=c(1),
 #'
 #' @inheritParams bootstrap.hankel
 #' @param debug Boolean. Enable debug output.
-#'
+#' @param deltat Integer. Time shift to be used to build the Hankel matrix
+#' @param submatrix.size Integer. Submatrix size to be used in build
+#'   of Hankel matrices. Submatrix.size > 1 is experimental.
+#' @param element.order Integer vector. specifies how to fit the \code{n} linearly ordered single
+#'    correlators into the correlator
+#'    matrix for submatrix.size > 1. \code{element.order=c(1,2,3,4)} leads to a matrix
+#'    \code{matrix(cf[element.order], nrow=2)}.
+#'    Double indexing is allowed.
```

Comment:
@urbach: what does "double indexing" mean?

Reply:
I guess that means you could give a triangular matrix and then specify `element.order = c(1, 2, 2, 3)`. If that is the case I'd rather call describe with a bit more prose and perhaps this example.